docs/README.md

```diff
@@ -44,8 +44,9 @@ To start a local Handsontable documentation server:
 
 From the `docs` directory, you can run the following npm scripts:
 
-* `npm run docs:start` – Starts a local documentation server at `localhost:8080/docs/`.
-* `npm run docs:start:no-cache` – Starts a local documentation server without cache.
+* `DOCS_VERSION=<semver.version> npm run docs:start:no-cache` – Starts a local documentation server, just for the <semver.version> documentation version.
+* `DOCS_VERSION=next npm run docs:start:no-cache` – Starts a local documentation server, just for the `next` documentation version.
+* `DOCS_FRAMEWORK=<javascript|react> npm run docs:start:no-cache` – Starts a local documentation server, just for the selected framework documentation version.
```

Comment:
```suggestion
* `DOCS_FRAMEWORK=<javascript|react> npm run docs:start:no-cache` – Starts a local documentation server, just for the selected framework documentation.
```

Let's not use the word "version" for anything other than the version number, because it becomes confusing.